caster-front/src/components/GraphPlayer.vue

```diff
@@ -29,17 +29,42 @@ import { useSendStreamVariableMutation } from "@/graphql";
 import { storeToRefs } from "pinia";
 import { usePlayerStore } from "@/stores/Player";
 
+import IntroScreen from "@/components/IntroScreen.vue";
+import IntroMoreInfo from "@/components/IntroMoreInfo.vue";
+import { PlayerState } from "@/models";
+import PlayerVisualizer from "@/components/PlayerVisualizer/PlayerVisualizer.vue";
+import PlayerBar from "@/components/PlayerBar/PlayerBar.vue";
+import EndScreen from "@/components/EndScreen.vue";
+
 interface DialogShow {
   show: boolean;
   loading: boolean;
   dialog: Extract<StreamSubscription["streamInfo"], { __typename: "Dialog" }>;
 }
 
 const props = defineProps<{
-  graph: Pick<Graph, "uuid" | "name">;
+  graph: Pick<
+    Graph,
+    | "uuid"
+    | "name"
+    | "aboutText"
+    | "displayName"
+    | "startText"
+    | "endText"
+    | "slugName"
+  >;
+  showDebug?: boolean;
```

Comment:
making `showDebug` optional via the `?` seems counter-intuitive for a boolean flag as it turns `true/false` to `true/false/undefined`.

Maybe defaulting it to `false` is a better way, see https://vuejs.org/guide/typescript/composition-api.html#typing-component-props

But I think it is not possible to apply partial defaults (e.g. we don't want to provide a default for `graph`).

In general only a remark for future development :)